### PR TITLE
AM-3028 Connect to v15 ORM AAT - v15 password, user and host taken fr…

### PR DESCRIPTION
**Pull request by mikebrownccd**
https://tools.hmcts.net/jira/browse/AM-3028
Connect to v15 ORM AAT

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] Does this PR introduce a breaking change

### CODE REVIEW

**Review by mattnayler — APPROVED** [review]
Approved as matches current ITHC.yaml

**Review by mattnayler — APPROVED** [review]
Approved based on match against:
* Demo PR from Branch `AM-2967_Connect_v15_Demo_ORM_Hardcoded`, see #26386.
* PerfTest PR from Branch `AM-3008_Connect_v15_ORM_Perftest`, see #25909 + #26217.

**Review by prashanthkumarnb — APPROVED** [review]

File: apps/am/am-org-role-mapping-service/aat.yaml [hunk 1]
```diff
@@ -5,8 +5,33 @@ metadata:
...
+              alias: ORG_ROLE_MAPPING_DB_PASSWORD
+            - name: judicial-topic-primary-send-listen-shared-access-key
+              alias: AMQP_JRD_SHARED_ACCESS_KEY_VALUE
       environment:
         AM_INFO: true
         APPLICATION_LOGGING_LEVEL: INFO
         SPRING_DATASOURCE_HIKARI_CONNECTIONTIMEOUT: 10000
         TESTING_SUPPORT_ENABLED: true
+        JUDICIAL_REF_APP_V2_ACTIVE: true
+        JUDICIAL_REF_APP_V2_FILTER_AUTHS_BY_APP_ID: true
```
Comment (mattnayler) [inline apps/am/am-org-role-mapping-service/aat.yaml:35]: The two V2 flags were retired in #28917.

Comment (mikebrownccd) [reply apps/am/am-org-role-mapping-service/aat.yaml]: these have been removed

**Review by mattnayler — APPROVED** [review]

**Pull request merged**